Classes/Modifier/AbstractRequestFieldModifier.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/yellowpages2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Yellowpages2\Modifier;
+
+use JWeiland\Yellowpages2\Middleware\ControllerActionsMiddleware;
+
+abstract class AbstractRequestFieldModifier implements RequestFieldModifierInterface
+{}
```

Comment:
Move behind class like done in TYPO3: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/core/Classes/Exception/SiteNotFoundException.php